components/lists/Signaling.tsx

```diff
@@ -24,6 +24,24 @@ type SearchProp = NativeStackNavigationProp<RootStackParamList, "Search">
 
 const Signaling = forwardRef<SignalingRef, SignalingProps>((_, ref) => {
   const navigation = useNavigation<SearchProp>()
+
+  async function fetchAllFriends() {
+    return api.get("/api/friend-signals").then((response) => response.data)
+  }
+
+  const loadFriends = async () => {
+    try {
+      const friends = await fetchAllFriends()
```

Comment:
what is this function doing and why is it not returning friends if it is fetching them